### PR TITLE
Add README.md

### PR DESCRIPTION
**Pull request by loociano**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# gba-tools
+
+Collection of tools for Gameboy Advance development.
```
Comment (fincs) [inline README.md]: Game Boy Advance

**Comment by WinterMute** [comment]
Will also need AM_INIT_AUTOMAKE([foreign]) in configure.ac and I think README.md will need added to EXTRA_DIST in Makefile.am to be included in release tarballs.

File: README.md (hunk 1 above)
Comment (loociano) [reply README.md]: Done, thanks!

**Comment by loociano** [comment]
Done. Let me know if I'm missing something else.

**Pull request merged**